### ISSUE TITLE
The plugin does not show the type hierarchy completely. For example, the hierarchy of the type `Widget` shows only a few of its children.

### ISSUE THREAD

**Issue by hcanyz** [flutter/flutter-intellij#6562]
I'm having the same problem. The plugin does not show the type hierarchy completely. For example, the hierarchy of the type `Widget` shows only a few of its children.

<img width="641" alt="image" src="https://user-images.githubusercontent.com/20122696/176391967-589f07b6-c57d-47b1-870b-59e8830bf974.png">

**UPDATE**

After some tests, I found it gives the wrong hierarchy tree, for example, the hierarchy of the type `Layer`:

<img width="492" alt="image" src="https://user-images.githubusercontent.com/20122696/176438819-7a454e5a-2329-4fd0-a03d-43c71f4fa3bf.png">

`TextureLayer`, `PictureLayer`, and `PlatformViewLayer` is not the child of the `ContainerLayer`, instead, it's `Layer`'s child.

_Originally posted by @NanoMichael in https://github.com/flutter/flutter-intellij/issues/5899#issuecomment-1169695961_

**Comment by hcanyz** [comment]
I also encountered the same problem

```
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 3.3.10, on Microsoft Windows [版本 10.0.19044.2364], locale zh-CN)
[√] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[√] Chrome - develop for the web
[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.4.2)
[√] Android Studio (version 2021.2)
[√] Android Studio (version 2021.3)
[√] VS Code (version 1.74.2)
[√] Connected device (4 available)
[√] HTTP Host Availability
```

**Comment by stevemessick** [comment]
That's a known problem in the Dart plugin.

**Comment by hcanyz** [comment]
> That's a known problem in the Dart plugin.

see https://youtrack.jetbrains.com/issue/WEB-54279/Dart-Issues-with-type-hierarchy-tool

**Comment by alexander-doroshko** [comment]
The problem [has been fixed](https://github.com/dart-lang/sdk/issues/50364) in the Dart Analysis Server. The fix is available starting from Dart SDK 2.19 (which currently has 'beta' status). No fixes needed in the Dart or Flutter plugin.
